src/devtools/client/webconsole/actions/toolbox.ts

```diff
@@ -18,7 +23,7 @@ export function highlightDomElement(grip: $FixTypeLater): UIThunkAction {
       return;
     }
 
-    const highlighter = window.gToolbox.getHighlighter();
+    const highlighter = window.gInspector.getHighlighter();
```

Comment:
we're slowly unwinding the globals... In all seriousness, we can probably do far better, but I havent taken the time to see what these Singletons are doing